package-lock.json

```diff
@@ -15898,13 +15913,12 @@
       "integrity": "sha1-99l9kt7mZl7F9toIx/ljytSyrJk="
     },
     "random-lib": {
-      "version": "2.1.0",
-      "resolved": "https://registry.npmjs.org/random-lib/-/random-lib-2.1.0.tgz",
-      "integrity": "sha1-PrOXD/J8Gvc8WIq5EHXY8KBDjfU=",
+      "version": "3.0.0",
```

Comment:
@riastradh-brave do you know why this diff is here?

Reply:
@diracdeltas It is because upstream bat-* packages were updated to use random-lib 3.0.0.  I filed PRs to remove those dependencies.  Once they are merged, I expect it will go away.  We can hold off until those PRs are merged and double-check if you like:

https://github.com/brave-intl/bat-client/pull/92
https://github.com/brave-intl/bat-ledger-spec/pull/27
https://github.com/brave-intl/bat-publisher/pull/41

Reply:
i think we should merge the upstream ones first so that this only requires one browser-laptop PR

Reply:
bat-ledger-spec appears to be not used in this repo; the others have been merged

Reply:
done in 2dae1d575e40b916f544e656c0b1d91a3fda9156